tutor/commands/compose.py

```diff
@@ -245,14 +245,22 @@ def start(
     services: list[str],
 ) -> None:
     command = ["up", "--remove-orphans"]
+    attach = len(services) == 1
```

Comment:
Nit: we can simplify the code slightly by writing:

 `attach = len(services) == 1 and not detach`

Then below:

```if detach: 
    command.append("-d")
...
if attach:
    ...
```

Reply:
That's a great suggestion, I've made the change.